models/models.js

```diff
@@ -8,6 +8,14 @@ exports.selectAllTopics = () => {
     })
 }
 
+exports.selectArticleById = (artId) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${artId}`)
```

Comment:
Shouldn't be using template literals here it allows for SQL injection.
Please look into sql parameterized query instead.